lib/cli/src/commands/create_exe.rs

```diff
@@ -433,21 +412,18 @@ pub(super) fn compile_pirita_into_directory(
         let atom_path = target_dir
             .join("atoms")
             .join(format!("{}.o", utils::normalize_atom_name(&atom_name)));
-        let header_path = match object_format {
-            ObjectFormat::Symbols => {
-                std::fs::create_dir_all(target_dir.join("include")).map_err(|e| {
-                    anyhow::anyhow!(
-                        "cannot create /include dir in {}: {e}",
-                        target_dir.display()
-                    )
-                })?;
+        let header_path = {
+            std::fs::create_dir_all(target_dir.join("include")).map_err(|e| {
+                anyhow::anyhow!(
+                    "cannot create /include dir in {}: {e}",
+                    target_dir.display()
+                )
+            })?;
 
-                Some(target_dir.join("include").join(format!(
-                    "static_defs_{}.h",
-                    utils::normalize_atom_name(&atom_name)
-                )))
-            }
-            ObjectFormat::Serialized => None,
+            Some(target_dir.join("include").join(format!(
```

Comment:
Minor nitpick: this is a big expression for a `Some(_)`, I'd move that out to a variable.